README.md

```diff
@@ -0,0 +1,105 @@
+# Vault HA Cluster
+
+This describes how to deploy a Vault HA cluster.
+
+## Prerequisites
+
+### System
+
+The [Terraform](https://www.terraform.io/) application must be installed on the system, as well as the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/).
+
+### Azure
+
+Some resources in [Azure](https://azure.microsoft.com/en-us/) are required: an Azure [subscription](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade), and a [registered application](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade). Both should be under the same Active Directory tenant.
```

Comment:
Should I add information about how to create subscriptions and register applications?

Reply:
Not needed I think, if you want to deploy a Vault cluster on Azure, you should have this knowledge